src/components/ChartView/ChartViewSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { fetchIncidentContacts } from '../../API-calls';
+
+const initialState = {
+  incidentContacts: [],
+  status: 'idle'
+}
+
+export const getIncidentContacts = createAsyncThunk(
+  'incidentContacts/getIncidentContacts',
+  async () => {
+    const response = await fetchIncidentContacts()
+    return response.data;
+  }
+)
+
+export const slice = createSlice({
+  name: 'incident-contacts',
+  initialState,
+  extraReducers: (builder) => {
+    builder
+      .addCase(getIncidentContacts.pending, state => {
+        state.status = 'loading'
+      })
+      .addCase(getIncidentContacts.fulfilled, (state, action) => {
+        state.status = 'idle'
+        state.incidentContacts = action.payload
+      })
+  }
+})
+
```

Comment:
This setup should allow us to compile the chart using data from the server once that endpoint is up and running. Hopefully we'll need to make only minimal changes around how the data is structured.